src/main/java/io/github/thebusybiscuit/slimefun4/utils/tags/SlimefunTag.java

```diff
@@ -251,7 +261,14 @@ public enum SlimefunTag implements Tag<Material> {
     /**
      * All materials that are affected by gravity.
      */
-    GRAVITY_AFFECTED_BLOCKS;
+    GRAVITY_AFFECTED_BLOCKS,
+
+    /**
+     * Different types of Tools.
+     */
+    PICKAXES,
+    SHOVELS,
+    AXES;
```

Comment:
It's slightly out of scope to what you're doing here, but while you're adding these three, perhaps it'd also be worth adding HOES? No use for the smelters ofc, but makes sense to have all four?

Reply:
yeah sure and ill add TOOLS for all of them combined

Reply:
Ah yeah, good point - bonus